lib/start-command.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+var debug = require('./debug')('start-command');
+var fs = require('fs');
+var path = require('path');
+
+module.exports = exports = resolveArgs;
+
+// for tests
+exports.fromPath = fromPath;
+exports.resolveFile = resolveFile;
+exports.resolveDir = resolveDir;
+exports.fromStart = fromStart;
+
+function resolveArgs(cwd, argv) {
+  var script = argv[2] || '.';
+  return fromPath(cwd, script);
+}
+
+function fromPath(cwd, script) {
+  var app = {
+    cwd: cwd,
+    path: script,
+    error: null,
+  };
+
+  try {
+    var stat = fs.statSync(path.resolve(app.cwd, app.path));
+  } catch (er) {
+    try {
+      app.path = require.resolve(path.resolve(app.cwd, app.path));
```

Comment:
This will hit if the path specified is a js file given without the `.js` extension, which `require()` and `node` will both honour.